collector/collector.go

```diff
@@ -196,7 +197,7 @@ func (c *Collector) addBuildAndJobMetrics(r *Result) error {
 				continue
 			}
 
-			pipeline := *build.Pipeline.Name
+			pipeline, _ := idna.ToASCII(*build.Pipeline.Name)
```

Comment:
Probably a good idea to check this error and return it if it occurs! Who knows what dragons live here :)